client/src/component/Dashboard/Dashboards/Bootcamp/views/BootcampProfile.jsx

```diff
@@ -1,15 +1,41 @@
-import React, { Component} from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios'
 
-class BootcampProfile extends Component {
-  state = {
-    
-  }
+const UserProfile = () => {
+  const [userInfo, setUserInfo] = useState({
+    id: "1",
+    email: '',
+    first_name: '',
+    last_name: '',
+    github_url: '',
+    linkedin_url: '',
+    bootcamp_name: ''
+
+  })
 
-  render() { 
-    return ( 
-      <h1>This is the Bootcamp Profile page!</h1>
-    );
+  const getUserInfo = async (id) => {
+    const response = await axios.get(`http://localhost:3000/profiles/id/${id}`)
+    setUserInfo(response.data)
```

Comment:
This information is already available in the Global State. if can be simplified by just pulling it using useSelector. if your thinking if for a company user for viewing a user Profile we will probably have a different component for that view in for the company user. We can discuss this tomorrow